doc/experiment.rst

```diff
@@ -1,327 +1,454 @@
+   * - a in-memory database (``Orchestrator``)
+     - the workload (``Model``)
+     - 
+   * - a workload (``Model``)
+     - 
+     - 
+
+Initialize
+----------
+.. compound::
+  To create a workflow, we *initialize* an ``Experiment`` object
+  once at the beginning of the Python driver script.
+  To create an Experiment, we specify a name
+  and the system launcher with which we will execute the driver script.
+  Here, we are running the example on a Slurm machine and as such will
+  set the `launcher` argument to `slurm`.
+
+  .. code-block:: python
+
+      from smartsim import Experiment
+      from smartsim.log import get_logger
+
+      # Initialize an Experiment
+      exp = Experiment("name-of-experiment", launcher="slurm")
+      # Initialize a SmartSim logger
+      smartsim_logger = get_logger("tutorial-experiment")
+
+  We also initialize a SmartSim logger. We will use the logger throughout the experiment
+  to monitor the entities.
+
+.. compound::
+  Next, we will launch a SmartSim in-memory database called an ``Orchestrator``.
+  To *initialize* an ``Orchestrator`` object, use the ``Experiment.create_database()``
+  function. We will create a single-sharded database and therefore will set
+  the argument `db_nodes` to 1. SmartSim will assign a `port` to the database
+  and detect your machine's `interface`.
+
+  .. code-block:: python
+
+      # Initialize an Orchestrator
+      database = exp.create_database(db_nodes=1)
+      # Create an output directory
+      exp.generate(database)
+
+  We use the ``Experiment.generate()`` function to create an
+  output directory for the database log files.
+
+.. compound::
+  Next, we create a workload within the experiment.
+  We begin by *initializing* a ``Model`` object.
+  To create a ``Model``, we must instruct SmartSim how we would
+  like to execute the workload by passing in a ``RunSettings``` object.
+  We create a RunSettings object using the
+  ``Experiment.create_run_settings()`` function.
+  We specify the executable to run and the arguments to pass to
+  the executable. The example workload is a simple `Hello World` program
+  that echos `Hello World` to stdout.
+
+  .. code-block:: python
+
+      settings = exp.create_run_settings("echo", exe_args="Hello World")
+      model = exp.create_model("hello_world", settings)
+
+  Notice above we creating the ``Model`` through the ``Experiment.create_model()``
+  function. We specify a `name` and the ``RunSettings`` object we created.
+
+
+Starting
+--------
+.. compound::
+  Next we will launch the components of the experiment (``Orchestrator`` and ``Model``) using functions
+  provided by the ``Experiment`` API. To do so, we will use
+  the ``Experiment.start()`` function and pass in the ``Orchestrator``
+  and ``Model`` instance previously created.
+
+  .. code-block:: python
+
+    # Launch the Orchestrator and Model instance
+    exp.start(database, model)
+    # log the status of the db
+    exp.get_status(database)
```

Comment:
Are these really logging the statuses or are they just return statuses in a programmatic way (not piped to log)?  If the latter, do we really need to initialize a logger in this experiment?  @ankona just dug into the logger, so he might have an opinion here.

Reply:
I would say no since the run settings I am passing in simply echos hello, we can instead demonstrate in a during tutorials